static/src/javascripts/projects/common/modules/identity/upsell/store/newsletters.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import {
+    getAllNewsletters,
+    getSubscribedNewsletters,
+} from 'common/modules/identity/api';
+import type { Followable } from 'common/modules/identity/upsell/consent-card/FollowCard';
+
+type Newsletter = {
+    id: string,
+    name: string,
+    description: string,
+};
+
+const set = (newsletterId: string, subscribed: boolean) => {
```

Comment:
We just have to implement this method once the API is there, and it should Just Work™